roles/matrix-backup-borg/tasks/setup_install.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- import_tasks: "{{ role_path }}/tasks/util/detect_existing_postgres_version.yml"
+  when: 'matrix_backup_borg_enabled|bool and matrix_backup_borg_postgresql_enabled|bool and matrix_postgres_backup_postgres_data_path != ""'
```

Comment:
`matrix_backup_borg_enabled|bool` should be unnecessary in `tasks/setup_install.yml`, because `tasks/setup_install.yml` only gets included (by `tasks/main.yml`) with the same condition (`matrix_backup_borg_enabled|bool`).